examples/dynamic_jsonpath.rs

```diff
@@ -13,19 +16,26 @@ async fn main() -> anyhow::Result<()> {
     // Equivalent to `kubectl get pods --all-namespace \
     // -o jsonpath='{.items[*].spec.containers[*].image}'`
     let field_selector = std::env::var("FIELD_SELECTOR").unwrap_or_default();
-    let jsonpath = format!(
-        "{}{}",
-        "$",
-        std::env::var("JSONPATH").unwrap_or_else(|_| ".items[*].spec.containers[*].image".into())
-    );
+    let jsonpath = {
+        let path = std::env::var("JSONPATH").unwrap_or_else(|_| ".items[*].spec.containers[*].image".into());
+        JsonPathInst::from_str(&format!("{}{}", "$", path.clone()))
```

Comment:
```suggestion
        format!("${path}").parse::<JsonPathInst>()
```

Also means you can get rid of the `FromStr` import

Reply:
Applied.